sdk/src/message.rs

```diff
@@ -162,6 +163,31 @@ pub struct Message {
     pub instructions: Vec<CompiledInstruction>,
 }
 
+impl Sanitize for Message {
```

Comment:
@jackcmay is this right?

Reply:
lgtm

Reply:
Should we also call `sanitize` on `self.account_keys`